manifests/Cromha Expansion.yaml

```
@@ -0,0 +1,13 @@
+name: Cromha Expansion Plugin
+authors: "@OcelotWalrus/Cromha, @Lumbar527"
+homepage: https://github.com/OcelotWalrus/Cromha-Expansion-plugin
+license: GPL-3.0-or-later
+version: v0.0.1
+shortDescription: Galaxy expansion plugin with various original stuff.
+description: Adds a new civilization to the Korath Space (now named the 'Empire Space'): The Cromha Empire. Also known as 'The Empire', it is a government that dirige a huge part of the north of the Milky Way. It also a very high advanced government, that has kind of remixed outfits and ships from other vanilla governments. They are human but don't directly have contact with human Merchants, they have contact with the Syndicate, the Republic but they don't allow them to enter their territory. Know more about the plugin in the wiki! (or test the game yourself)
```

```suggestion
description: "Adds a new civilization to the Korath Space (now named the 'Empire Space'): The Cromha Empire. Also known as 'The Empire', it is a government that dirige a huge part of the north of the Milky Way. It also a very high advanced government, that has kind of remixed outfits and ships from other vanilla governments. They are human but don't directly have contact with human Merchants, they have contact with the Syndicate, the Republic but they don't allow them to enter their territory. Know more about the plugin in the wiki! (or test the game yourself)"
```
Again, quote the entire thing (this time with quotes that arent used in the text itself)

---

yes